deploy/kubernetes/console/templates/deployment.yaml

```diff
@@ -182,8 +182,12 @@ spec:
         - name: SKIP_SSL_VALIDATION 
           value: {{default "true" .Values.uaa.skipSSLValidation | quote}}
         {{- end }}
-        {{- if .Values.env.DOMAIN }}  
-        - name: X_COOKIE_DOMAIN
+        {{- if .Values.console.cookieDomain }}{{ if ne .Values.console.cookieDomain "-" }}
```

Comment:
both `if`s can be merged
```
{{- if and .Vales.console.cookieDomain (ne .Values.console.cookieDomain "-") }}

Reply:
@irfanhabib - The logic I want is: if cookieDomain is set, use it, unless it is set to '-' in which case do not set it, otherwise default to env.DOMAIN - I don't think what you propose will do that.